src/button-dropdown/__tests__/button-dropdown.test.tsx

```diff
@@ -100,6 +101,43 @@ const items: ButtonDropdownProps.Items = [
           expect(renderedItems.length).toEqual(0);
           expect(wrapper.findOpenDropdown()).toBe(null);
         });
+
+        describe('disabled with reason', () => {
+          jest.spyOn(ReactDOM, 'createPortal').mockImplementation((element: any) => element);
```

Comment:
Tooltip component uses react portal. This mock ensures that tooltips is rendered within root element to make using test-utils possible

Reply:
This won't work. We cannot ask all consumers of this feature to do the same in their tests.

Check how other components implement portal handling in test utils and do the same here.

Reply:
done for all components
changed `findDisabledReason` implementation to make it access tooltip portal via document root:

```javascript
findDisabledReason(): ElementWrapper | null {
    return createWrapper().find(`.${styles['disabled-reason-tooltip']}`);
 }
```

Reply:
Great thanks. I am leaving this PR now